AutoCompletion.podspec

```diff
@@ -0,0 +1,13 @@
+Pod::Spec.new do |s|
+s.name         = "AutoCompletion"
+s.version      = "1.2.1"
+s.summary      = "Text auto completion framework."
+s.description  = "Library can be used via cocoapods in any project."
+s.homepage     = "https://github.com/3pillarlabs/ios-autocomplete/tree/AutoCompleteFramework"
```

Comment:
`https://github.com/3pillarlabs/ios-autocomplete`should be the home page.

Reply:
Updated.